core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -355,15 +355,15 @@
   public static final PropertyKey WORKER_NETWORK_NETTY_READER_BUFFER_SIZE_PACKETS =
       create(Name.WORKER_NETWORK_NETTY_READER_BUFFER_SIZE_PACKETS, 16);
   public static final PropertyKey WORKER_NETWORK_NETTY_BLOCK_READER_THREADS_MAX =
-      create(Name.WORKER_NETWORK_NETTY_BLOCK_READER_THREADS_MAX, 128);
```

Comment:
@calvinjia I've increased this so that it can handle high concurrent ufs read (thundering herd problem).